src/components/PackageList/PackageList.tsx

```diff
@@ -1,4 +1,4 @@
-import * as React from 'react';
+import React, { Fragment, ReactElement } from 'react';
```

Comment:
Maybe we should discuss about some standards, but I am trying to keep the imports together, like you did here : 
![image](https://user-images.githubusercontent.com/29228205/67668022-6bfdec00-f96f-11e9-9eca-f184947f8bf5.png)

Reply:
yes! is there a weekly/monthly call or is discord the best forum for discussions?

Reply:
yes you can always contact us on verdaccio discord chat 😃This week I'm a little absent, but surely the other members will be online to help you

Reply:
sounds good 👍